Kite SDK/PSPrintSDK/OLKiteABTesting.m

```diff
@@ -171,10 +172,25 @@ - (void)setupHidePriceTest{
                                 }];
 }
 
+- (void)setupOfferPayPalTest{
+    NSDictionary *experimentDict = [[NSUserDefaults standardUserDefaults] objectForKey:kOLKiteABTestOfferPayPal];
+    if (!experimentDict) {
+        experimentDict = @{@"Yes" : @0, @"No" : @1};
```

Comment:
Cool, I’ll make it 0.5/0.5. I think we should still get a feel for how it affects conversion.

> On 23 Jul 2015, at 15:30, Konstadinos Karayannis <notifications@github.com> wrote:
> 
> In Kite SDK/PSPrintSDK/OLKiteABTesting.m <https://github.com/OceanLabs/iOS-Print-SDK/pull/202#discussion_r35326872>:
> 
> > @@ -171,10 +172,25 @@ - (void)setupHidePriceTest{
> >                                  }];
> >  }
> >  
> > +- (void)setupOfferPayPalTest{
> > +    NSDictionary *experimentDict = [[NSUserDefaults standardUserDefaults] objectForKey:kOLKiteABTestOfferPayPal];
> > +    if (!experimentDict) {
> > +        experimentDict = @{@"Yes" : @0, @"No" : @1};
> I put that in deliberately because I thought that not having PayPal by default (even 0.5 of the time) carries some inherent risk that all our other AB tests don't. But I don't have arguments other than that so feel free to change it.
> … <x-msg://20/#>
> —
> Reply to this email directly or view it on GitHub.
> 
>  <https://github.com/OceanLabs/iOS-Print-SDK/pull/202#discussion_r35326471> <https://github.com/0> <https://github.com/1> <https://github.com/OceanLabs/iOS-Print-SDK/pull/202/files#r35326471>
> —
> Reply to this email directly or view it on GitHub <https://github.com/OceanLabs/iOS-Print-SDK/pull/202/files#r35326872>.
>